dockers/docker-dhcp-relay/start.sh

```diff
@@ -1,5 +1,11 @@
 #!/usr/bin/env bash
 
+if [ "${RUNTIME_OWNER}" == "" ]; then
```

Comment:
i am not familiar with this RUNTIME_OWNER concept, is this SET_OWNER?

Reply:
Nope. This is the 'who' running the container. As name suggests, runtime owner of this container instance.

In case of 'local' "docker create" in /usr/bin/<feature>.sh will create the container  instance with "-e RUNTIME_OWNER=local" (changes in   [files/build_templates/docker_image_ctl.j2](https://github.com/Azure/sonic-buildimage/pull/5421/files#diff-130e5ab75471398db06b73a4c6e4b56517f42e3c1f17870b5309620fc0e2fafb)).

When started by kube, the manifest would/*expected-to* set the environment variable as below.
        env:
        - name: RUNTIME_OWNER
          value: "kube"

When the container is started, it passes this RUNTIME_OWNER as the current to container_state, which it sets in current_owner in STATE_DB.

Being hardcoded in "/usr/bin/<name>.sh", the local instance is assured to get this ENV var. So default to Kube, if in case missing.